test/pylib/scylla_cluster.py

```diff
@@ -503,6 +503,14 @@ async def stop(self) -> None:
         if not self.cmd:
             return
 
+        # Dump the profile if exists and supported by the API.
+        try:
+            api = ScyllaRESTAPIClient()
+            await api.dump_profile(self.host_id)
```

Comment:
I think you have a typo here - the function needs an host **IP** and you give it a host **ID**.
I suspect this doesn't even work, and it was just ignored (you got to the silent "pass" case below) and you missed it in your testing :-(

Reply:
Will validate

Reply:
You were right.
I fixed it.